src/sentry/integrations/slack/utils/channel.py

```diff
@@ -97,14 +99,33 @@ def validate_channel_id(name: str, integration_id: int | None, input_channel_id:
     """
     # The empty string should be converted to None
     payload = {"channel": input_channel_id or None}
-    client = SlackClient(integration_id=integration_id)
-    try:
-        results = client.get("/conversations.info", params=payload)
-    except ApiError as e:
-        if e.text == "channel_not_found":
-            raise ValidationError("Channel not found. Invalid ID provided.")
-        logger.info("rule.slack.conversation_info_failed", extra={"error": str(e)})
-        raise IntegrationError("Could not retrieve Slack channel information.")
+
+    logger_params = {
+        "integration_id": integration_id,
+        "channel_name": name,
+        "input_channel_id": input_channel_id,
+    }
```

Comment:
Nit: Are we trying to follow a convention of declaring `logger_params` as a local variable? I don't think it's necessary unless the same params are logged in multiple places.